src/App.js

```diff
@@ -1,32 +1,33 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import './App.css';
 import jwtDecode from 'jwt-decode';
 import axios from 'axios';
 
-// Redux
-import { Provider } from 'react-redux';
-import store from './redux/store';
-import { SET_AUTHENTICATED } from './redux/types';
-import {
-  logoutUserAction,
-  getUserDataAction,
-} from './redux/actions/userActions';
-
-// Components
-import Navbar from './components/Navbar';
-import AuthRoute from './util/AuthRoute';
-
 // Pages
 import home from './pages/home';
 import login from './pages/login';
 import signup from './pages/register';
+import user from './pages/user';
+
+// Components
+import Navbar from './components/Navbar';
+import AuthRoute from './util/AuthRoute';
 
 // MUI
 import deepPurple from '@material-ui/core/colors/deepPurple';
 import { ThemeProvider } from '@material-ui/core/styles';
 import createMuiTheme from '@material-ui/core/styles/createMuiTheme';
 
+// Redux
+import { Provider } from 'react-redux';
+import store from './redux/store';
+import { SET_AUTHENTICATED } from './redux/types';
+import {
+  logoutUserAction,
+  getUserDataAction,
+} from './redux/actions/userActions';
```

Comment:
I don't see where these are being used?

Reply:
They are being used in handling the user's token that's stored in their local storage. 
![image](https://user-images.githubusercontent.com/7955498/72594075-70c1a480-38bb-11ea-87b8-704a463c077f.png)